### PR TITLE
Feature: support .invalidate()

### PR DESCRIPTION
**Pull request by niklasR**
With this feature, we can invalidate certain keys manually without waiting for the expiry of the TTL.

### CODE REVIEW

File: lib/ceych.js [hunk 1]
```diff
@@ -83,4 +84,12 @@ Ceych.prototype.disableCache = function() {
   this.cache.stop();
 };
 
+Ceych.prototype.invalidate = function(fn) {
+  const args = Array.from(arguments);
+  const cb = args.pop();
```
Comment (nspragg) [inline lib/ceych.js]: Probably a programming error if the callback is absent, so do we need to assert for that? 

`memorise` uses `getFunctionCallback` for this reason.

File: test/lib/ceych.js [hunk 2]
```diff
@@ -154,4 +155,99 @@ describe('ceych', () => {
...
+  describe('.invalidate', () => {
+    it('invalidates the cache entry', (done) => {
+      const func = ceych.wrap(wrappable);
+
+      func((err, result) => {
+        assert.ifError(err);
+        assert.equal(result, 1);
+        sinon.assert.calledOnce(wrappable);
+
+        // Invalidate the cache entry
```
Comment (nspragg) [inline test/lib/ceych.js]: You can probably drop the comments with the exception of the more complex test (like the last one).

**Comment by nspragg** [comment]
👍 

Considerations: 
1) Add test for invalidate with missing `cb` (optional) 
2) Running a local integration test (Recommended)

**Comment by niklasR** [comment]
1. Added test (and fixed it.. good suggestion)

2. Will do when integrating 👍

**Pull request merged**